ui/src/testing/stubs/disks-api-stub.ts

```diff
@@ -8,57 +8,53 @@ import {
   ListDisksResponse,
 } from 'generated/fetch';
 
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export const stubDisk = (): Disk => ({
   size: 1000,
-  diskType: DiskType.Standard,
+  diskType: DiskType.STANDARD,
   isGceRuntime: true,
   name: 'stub-disk',
   blockSize: 1,
 });
 
 export const mockJupyterDisk = (): Disk => ({
   size: 1000,
-  diskType: DiskType.Standard,
+  diskType: DiskType.STANDARD,
   isGceRuntime: true,
   name: 'mock-disk1',
   blockSize: 1,
-  status: DiskStatus.Ready,
+  status: DiskStatus.READY,
   appType: null,
   creator: '"evrii@fake-research-aou.org"',
   createdDate: '2023-05-22T18:55:10.108838Z',
 });
 
 export const mockCromwellDisk = (): Disk => ({
   size: 1000,
-  diskType: DiskType.Standard,
+  diskType: DiskType.STANDARD,
   isGceRuntime: false,
   name: 'mock-disk2',
   blockSize: 1,
-  status: DiskStatus.Ready,
+  status: DiskStatus.READY,
   appType: AppType.CROMWELL,
   creator: '"evrii@fake-research-aou.org"',
   createdDate: '2023-05-22T18:55:10.108838Z',
 });
 
 export const mockRStudioDisk = (): Disk => ({
   size: 1000,
-  diskType: DiskType.Standard,
+  diskType: DiskType.STANDARD,
   isGceRuntime: false,
   name: 'mock-disk3',
   blockSize: 1,
-  status: DiskStatus.Ready,
+  status: DiskStatus.READY,
   appType: AppType.RSTUDIO,
   creator: '"evrii@fake-research-aou.org"',
   createdDate: '2023-05-22T18:55:10.108838Z',
 });
 
 export class DisksApiStub extends DisksApi {
   constructor(public disk?: Disk) {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.